Documentation/ceph-advanced-configuration.md

```diff
@@ -30,6 +31,37 @@ The Kubernetes based examples assume Rook OSD pods are in the `rook-ceph` namesp
 If you run them in a different namespace, modify `kubectl -n rook-ceph [...]` to fit
 your situation.
 
+## Using alternate namespaces
+
+If you wish to deploy the Rook Operator and/or Ceph clusters to namespaces other than the default
+`rook-ceph`, the manifests are commented to allow for easy `sed` replacements. Change
+`ROOK_CLUSTER_NAMESPACE` to tailor the manifests for additional Ceph clusters. You can choose
+to also change `ROOK_OPERATOR_NAMESPACE` to create a new Rook Operator for each Ceph cluster (don't
+forget to set `ROOK_CURRENT_NAMESPACE_ONLY`), or you can leave it at the same value for every
+Ceph cluster if you only wish to have one Operator manage all Ceph clusters.
+
+This will help you manage namespaces more easily, but you should still make sure the resources are
+configured to your liking.
+
+```sh
+cd cluster/examples/kubernetes/ceph
+
+export ROOK_OPERATOR_NAMESPACE="rook-ceph"
+export ROOK_CLUSTER_NAMESPACE="rook-ceph"
+
+sed -i.bak \
+    -e "s/\(.*\):.*# namespace:operator/\1: $ROOK_OPERATOR_NAMESPACE # namespace:operator/g" \
+    -e "s/\(.*\):.*# namespace:cluster/\1: $ROOK_CLUSTER_NAMESPACE # namespace:cluster/g" \
+    -e "s/\(.*serviceaccount\):.*:\(.*\) # serviceaccount:namespace:operator/\1:$ROOK_OPERATOR_NAMESPACE:\2 # serviceaccount:namespace:operator/g" \
+    -e "s/\(.*serviceaccount\):.*:\(.*\) # serviceaccount:namespace:cluster/\1:$ROOK_CLUSTER_NAMESPACE:\2 # serviceaccount:namespace:cluster/g" \
+    -e "s/\(.*\): [-_A-Za-z0-9]*\.\(.*\) # driver:namespace:operator/\1: $ROOK_OPERATOR_NAMESPACE.\2 # driver:namespace:operator/g" \
+    -e "s/\(.*\): [-_A-Za-z0-9]*\.\(.*\) # driver:namespace:cluster/\1: $ROOK_CLUSTER_NAMESPACE.\2 # driver:namespace:cluster/g" \
+  common.yaml operator.yaml cluster.yaml # add other files or change these as desired for your config
```

Comment:
What about all the other yamls where the replacement is necessary such as CRs and storage classes with the csi driver name? We can do this in a follow up PR